### PR TITLE
Force record ids to be integers.

### PR DESCRIPTION
**Pull request by qubixes**
Right now floating point numbers as record id's or missing record id's might cause trouble.

### CODE REVIEW

**Review by J535D165 — APPROVED** [review]

File: asreview/io/utils.py [hunk 1]
```diff
@@ -121,10 +121,11 @@ def standardize_dataframe(df, column_spec={}):
         else:
             try:
                 df['record_id'] = pd.to_numeric(df['record_id'])
+                df['record_id'] = df['record_id'].astype(int)
```
Comment (J535D165) [inline asreview/io/utils.py:124]: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.to_numeric.html#pandas.to_numeric 

Use downcast argument?

Comment (qubixes) [reply asreview/io/utils.py:124]: I tried that, but it didn't work...

Comment (J535D165) [reply asreview/io/utils.py:124]: Oke.

**Comment by J535D165** [comment]
Can you explain why record_ids are converted to integers? Why not accept any unique, complete list of record_ids?

**Comment by qubixes** [comment]
It seems more convenient than strings to me. But accepting them as strings is possible (but requires modification of the code).

**Comment by J535D165** [comment]
For example, the CORD-19 database has MDF5 identifiers. I think we should not try to convert record ids into integers.

**Comment by qubixes** [comment]
They will be ignored. I think there is a case to be made to use asreview_record_id, though that should be adjusted after this PR. (I don't think it matters all that much internally, but it might save some confusion.)

**Comment by J535D165** [comment]
Oke, fine for me now. Will merge after a unit test is available (for both negative and missing record_ids in the `ASReviewData` class).

**Comment by qubixes** [comment]
I have added the two missing unit tests. There is some commented out code that might be useful if we change the warning logs to actual warnings.

**Pull request closed without merging**